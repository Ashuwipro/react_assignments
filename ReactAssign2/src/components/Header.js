export default function Header() {
  const styles = {
    color: "red",
    fontWeight: "bold",
    fontSize: "30px",
    marginLeft: "30px",
    marginTop: "20px",
    marginBottom: "20px",
  };
  return (
    <div style={styles}>
      FED React : Assignment2 : Multiple Components, JSON
    </div>
  );
}
