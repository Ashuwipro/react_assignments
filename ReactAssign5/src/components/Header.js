export default function Header() {
  const styles = {
    color: "red",
    fontWeight: "bold",
    fontSize: "30px",
    marginLeft: "30px",
    marginTop: "20px",
    marginBottom: "20px",
  };
  return (
    <div style={styles}>
      FED React : Assignment5 : Functional Components, Hooks
    </div>
  );
}
