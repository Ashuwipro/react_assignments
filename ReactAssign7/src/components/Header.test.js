import Header from "./Header";
import { shallow } from "enzyme";

test("should render correct heading", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper.find("div").text()).toEqual(
    "FED React : Assignment7 : Multiple Components, JSON"
  );
});
