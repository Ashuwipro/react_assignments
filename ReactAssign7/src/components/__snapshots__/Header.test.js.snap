// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correct heading 1`] = `ShallowWrapper {}`;
